bridges/snowbridge/pallets/ethereum-client/src/lib.rs

```diff
@@ -179,6 +179,10 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type NextSyncCommittee<T: Config> = StorageValue<_, SyncCommitteePrepared, ValueQuery>;
 
+	/// The last period where the next sync committee was updated for free.
+	#[pallet::storage]
+	pub type LatestFreeSyncCommitteeUpdatePeriod<T: Config> = StorageValue<_, u64, ValueQuery>;
```

Comment:
Would it make sense to rename this to `LatestSyncCommitteeUpdatePeriod` ? Looks like it's set even if the sync committee update is not free.

Reply:
That's a good point! :) https://github.com/paritytech/polkadot-sdk/pull/5671/commits/6e4f99daf4d2ffeccefbbd8a4ab3f1eaabe077a7